text/0006-BundleContext-Install-API.md

```diff
@@ -0,0 +1,239 @@
+- Start Date: 2020-06-09
+- RFC PR: (in a subsequent commit to the PR, fill me with the PR's URL)
+- CppMicroServices Issue: (fill me with the URL of the CppMicroServices issue that necessisated this RFC, otherwise leave blank)
+
+# Add BundleManifest Injection Capabilities 
+
+## Summary
+
+Provide a api to initialize the bundle registry with a bundle manifest during the BundleContext::InstallBundles operation by providing an optional argument containing a manifest.
+
+## Motivation
```

Comment:
somewhere it should be noted that this design attempts to follow the OSGi spec by implementing an equivalence of https://docs.osgi.org/specification/osgi.core/7.0.0/framework.api.html#org.osgi.framework.BundleContext.installBundle-String-InputStream-

Reply:
Added.